scipy/optimize/setup.py

```diff
@@ -116,6 +116,10 @@ def configuration(parent_package='', top_path=None):
     config.add_subpackage('_shgo_lib')
     config.add_data_dir('_shgo_lib')
 
+    # HiGHS linear programming libraries and extensions
+    config.add_subpackage('_highs')
+    config.add_data_dir('_highs')
```

Comment:
This is incorrect, it causes all source code in this directory to be installed. This should be either removed, or if the `*.pxd` files or some other subset of files is needed, those should selectively be added.

Reply:
See https://github.com/mdhaber/scipy/pull/46; `*.pxd` are added as data files instead of the entire directory tree